keyboards/keyten/kt60hs_t/info.json

```
@@ -0,0 +1,3 @@
+{
+    "maintainer": "key10iq"
+}
```

You can move more things to the common `info.json`, e.g.:
```suggestion
{
    "manufacturer": "keyten",
    "keyboard_name": "kt60HS-T",
    "maintainer": "key10iq",
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "nkro": true
    },
    "usb": {
        "vid": "0xEB69"
    }
}
```
Then you can remove these entries from `keyboard.json` files in subfolders (I can't put the removal of them as suggestions at least for the `v1` case, so you would need to do that part yourself).

Technically you can even move the layout definitions here if they are exactly the same, but this may need to be reverted if you ever want to do a V3 with some different matrix wiring.